proteus/cnumericalFlux.pyx

```diff
@@ -3310,10 +3286,10 @@ def calculateDiffusionMatrixSplittings_LDG_sd(int aSplit,
                            np.ndarray aHat,
                            np.ndarray aTilde):
      ccalculateDiffusionMatrixSplittings_LDG_sd( aSplit,
-                            nElements_global,
-                            nElementBoundaries_element,
-                            nQuadraturePoints_element,
-                            nQuadraturePoints_elementBoundary,
+					         q_a.shape[0],
```

Comment:
@wacyyang here are the kind of problems that need checking for on cnumericalFlux.pyx

Reply:
@cekees I have compared those change to https://github.com/erdc/proteus/tree/fa592f1be42d5c2cf3c1c5053d023eb3271842a5. And the interface of those 6 functions are correct now.